dotCMS/core
# #23600 Use same thumbnail size everywhere part 1

### Proposed Changes
* Moves us to `/dA/{inode}/500w/20q` as our default for thumbnail generation
* Updates listing view thumbnails to use the `dot-thumbnail` component.
* Updated the relationship listing to use the same resolution thumbnail as the `dot-thumbnail` component - meaning it won't have to be regenerated by dotCMS
* Updated the content pallet listing to use the same resolution thumbnail as the `dot-thumbnail` component - meaning it won't have to be regenerated by dotCMS

### To Test
- In a new install, you should be able to switch between card and listing views without a hiccup (you can delete your `/assets/dotGenerated` folder to test
- You should be able to see thumbnails in the relationship tabe
- You should be able to see thumbnails in the content pallet (from edit mode)
- If you upload a video, you should see a video thumbnail in both the listing and card screens
- listing thumbs should be a bit bigger

### Screenshots
Original             |  Updated
:-------------------------:|:-------------------------:
![master](https://github.com/dotCMS/core/assets/934364/85446c18-a98d-4e3e-b6c7-e989e426725d)  | ![with-changes](https://github.com/dotCMS/core/assets/934364/d7ad7de6-0d38-4f55-9f54-3ad52c6ebf89)
